src/content/history/index.md

```diff
@@ -194,7 +194,7 @@ The Muir Glacier fork introduced a delay to the [difficulty bomb](/glossary/#dif
 The Istanbul fork:
 
 - Optimised the [gas](/glossary/#gas) cost of certain actions in the [EVM](/developers/docs/ethereum-stack/#ethereum-virtual-machine).
-- Improved denial-of-service attack resilience.
+- Improved denial of service attack resilience.
```

Comment:
We should keep this as is. Hyphenating denial-of-service is the most common usage: https://en.wikipedia.org/wiki/Denial-of-service_attack

```suggestion
- Improved denial-of-service attack resilience.
```